static/js/chain.js

```diff
@@ -25,3 +25,35 @@ function showHide(show, hide) {
     $(show).each(function(){ $(this).prop('disabled', false).css('opacity', 1.0) });
     $(hide).each(function(){ $(this).prop('disabled', true).css('opacity', 0.5) });
 }
+
+function controlOp(mode){
+    let op = $('#operations option:selected').attr('value');
+    $.ajax({
+            url: `/op/control`,
+            type: 'post',
+            data: {'id': op, 'mode': mode},
+            success: function (data) {
+                getOpState();
+            }
+        });
+}
+
+function getOpState() {
+    let op = $('#operations option:selected').attr('value');
+    $.ajax({
+            url: `/op/control`,
+            type: 'post',
+            data: {'id': op, 'mode': 'state'},
+            success: function (data) {
+                if (data['result'] === 'PAUSED'){
+                    $('#control-play').css('display','');
+                    $('#control-pause').css('display','none');
+                }
+                if (data['result'] === 'RUNNING') {
+                    $('#control-play').css('display','none');
+                    $('#control-pause').css('display','');
+                }
+                document.getElementById('control-state').innerHTML = data['result']
```

Comment:
These AJAX requests should adhere to the front-to-back-end call structure we've agreed to: https://confluence.mitre.org/display/CAL/CALDERA+-+Chain+Mode%3A++Front-end+to+Back-end+API+Calls

Reply:
Let's update the JS function to include a Default endpoint & override ability like:
`function restRequest(type, data, callback, endpoint='/plugin/chain/rest'){//code}`